### ISSUE TITLE
Can you share your ttt file?

### ISSUE THREAD

**Issue by wyd0817** [tapabratadey/disaster_robot#1]
I'm trying to write lua code via subscript to let the simulation depict the graph, but my simulation always complains as follows
```
4: attempt to call field 'addGraphStream' (a nil value)
    stack traceback:
        [string "Graph@childScript"]:4: in function <[string "Graph@childScript"]:1>
```
Can I see your ttt file as a reference please?
Good wishes

**Comment by tapabratadey** [comment]
Hi, I've only made some tweaks to the bubbleRob bot, take a look into it, it might help you. Your addGraphStream() params are all valid?

**Comment by wyd0817** [comment]
Hi, thank you for your reply.
I think all my params are valid because I am just using[ the example from the official website](https://www.coppeliarobotics.com/helpFiles/en/graphs.htm).

I checked my I put bubbleRob bot, but he has no programming for graph, so I copied [your lua code](https://github.com/tapabratadey/disaster_robot/blob/bc141e5be23051987a8243ccc576df1443e01a22/disaster_robot.lua
) into a subscript of my bubbleRob bot and added a person and sensor, and it showed the same error
`18: attempt to call field 'addGraphStream' (a nil value)
    stack traceback:
           [string "bubbleRob@childScript"]:18: in function <[string "bubbleRob@childScript"]:1>`
It's a bit strange, I'm confused. So if you don't care, share your ttt file with me, I would be very grateful!

**Comment by tapabratadey** [comment]
No worries! I just pushed the ttt file lmk if it helps! Good luck!

**Comment by wyd0817** [comment]
> No worries! I just pushed the ttt file lmk if it helps! Good luck!

Thanks for the reply and sharing.
I downloaded [it](https://github.com/tapabratadey/disaster_robot/blob/main/disaster_scene_save_person.ttt) and ran it in my Coppeliasim. It also complains about a nil value of the parameter.
```
[bubbleRob@childScript:error]   18: attempt to call field 'addGraphStream' (a nil value)
    stack traceback:
        [string "bubbleRob@childScript"]:18: in function <[string "bubbleRob@childScript"]:1>
```
What is the version of Coppeliasim you are using?

**Comment by tapabratadey** [comment]
![Screen Shot 2021-04-28 at 4 25 32 PM](https://user-images.githubusercontent.com/12387917/116468131-7b991200-a83e-11eb-9400-d0a2569a1007.png)

**Comment by wyd0817** [comment]
My previous version was V4.1.0. When I updated my Coppeliasim to V4.2.0 , both your ttt file and my ttt file worked fine.
When I read Coppeliasim's [release documentation](https://www.coppeliarobotics.com/helpFiles/en/versionInfo.htm#coppeliaSim4.2.0) I understood why. He added the adjustment of creation curves via API functions only in the latest version. And I ignored him.
`Since this release, graph streams and curves created via the GUI are deprecated in favor of streams/curves created and manipulated via new API functions. This allows a much more flexible and precise control of graphs. Read more about graphs here.`
Thank you for your selfless sharing, thanks

Cheers!
